5.1/5.1.0/debian/buildout.cfg

```diff
@@ -0,0 +1,29 @@
+[buildout]
+extends =
+  buildout-base.cfg
+
+extensions =
+  buildout.wheel
+
+effective-user = plone
+buildout-user = plone
+var-dir=/data
+user=admin:admin
+parts +=
+  zeo
+
+[client1]
+recipe =
+
+[zeo]
+<= zeoserver_base
+recipe = plone.recipe.zeoserver
+zeo-address = 8080
+
+[versions]
+setuptools =
+zc.buildout =
+
+buildout.wheel = 0.2.0
+plone.recipe.command = 1.1
+plone.recipe.precompiler = 0.6
```

Comment:
Why we are adding the buildout.cfg ?
We could also do a wget in the Dockerfile to get the latest from the UI installer repo.

By doing so we would make sure that we are always in sync and use the same buildout.cfg :)

Reply:
I used the `buildout.cfg` from UI as `buildout-base.cfg`. This one is for backward compatibility and to avoid tweaks within Dockerfile. 

These versions are not pinned anywhere, that's why I pinned them here.

Reply:
We download UI and get from there `base_skeleton`

Now, in order not to do `sed` and other tweaks within Dockerfile and also to keep backward compatibility, I renamed UI `buildout.cfg` to `buildout-base.cfg` and do the tweaks within our custom `buildout.cfg` that extends `buildout-base.cfg`.